modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,97 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify if terms from a related custom object (child) with a system object (parent) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromSystemObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an object with a field is created") {
```

Comment:
`Given a custom object with a text field`

Be specific with what you're adding